dev.py

```diff
@@ -52,8 +52,11 @@ def install_pip():
     tmp = tempfile.mkdtemp(prefix='buildout-dev-')
     try:
         get_pip = os.path.join(tmp, 'get-pip.py')
+        GET_PIP_URL = 'https://bootstrap.pypa.io/get-pip.py'
+        if sys.version_info < (3, ):
+            GET_PIP_URL = 'https://bootstrap.pypa.io/2.7/get-pip.py'
```

Comment:
I think we also need to special-case Python 3.5 in a similar way.

Reply:
Now that you say it... I just wonder why both CI and my local test run did not show any problems. But let's do it anyway. Looks like we have to support 3.5 until the end of 2022 ( https://www.zope.org/developer/roadmap.html ). Updated with next commit.